package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "madie-models",
+  "name": "@madie/madie-models",
```

Comment:
Not sure if @madie is needed in the lock file as well but running `npm install` will update the lock file.